### ISSUE TITLE
Mac installation fails: readRDS error

### ISSUE THREAD

**Issue by pontikos** [jalvesaq/VimCom#22]
I am not sure if this is an issue with my R installation.
This is the version of R I am running:

```
R version 3.1.2 (2014-10-31) -- "Pumpkin Helmet"
Copyright (C) 2014 The R Foundation for Statistical Computing
Platform: x86_64-apple-darwin13.4.0 (64-bit)
```

and this is the error I get:

```
> devtools::install_github('jalvesaq/VimCom')
Downloading github repo jalvesaq/VimCom@master
Installing vimcom
'/Library/Frameworks/R.framework/Resources/bin/R' --vanilla CMD INSTALL  \
  '/private/var/folders/rc/_l4njjr528z8t6bk_kfzdg4h0000gn/T/RtmpTjIvVL/devtools763f27ddbfb3/jalvesaq-VimCom-89c6c01'  \
  --library='/Library/Frameworks/R.framework/Versions/3.1/Resources/library'  \
  --install-tests

* installing *source* package ‘vimcom’ ...
configure: creating ./config.status
config.status: creating src/Makevars
** libs
clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG -DNO_X_CLIENTSERVER  -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include    -fPIC  -Wall -mtune=core2 -g -O2  -c vimcom.c -o vimcom.o
clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG -DNO_X_CLIENTSERVER  -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include    -fPIC  -Wall -mtune=core2 -g -O2  -c vimremote.c -o vimremote.o
clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG -DNO_X_CLIENTSERVER  -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include    -fPIC  -Wall -mtune=core2 -g -O2  -c vimthings.c -o vimthings.o
clang -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/lib -o vimcom.so vimcom.o vimremote.o vimthings.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
(cd apps; make)
cc -std=gnu99 -fPIC -O2 -c vimr.c -o vimr.o
cc -std=gnu99 -shared -o libVimR.so vimr.o
installing via 'install.libs.R' to /Library/Frameworks/R.framework/Versions/3.1/Resources/library/vimcom
** R
** preparing package for lazy loading
Error in readRDS(pfile) : error reading from connection
ERROR: lazy loading failed for package ‘vimcom’
* removing ‘/Library/Frameworks/R.framework/Versions/3.1/Resources/library/vimcom’
Error: Command failed (1)
```

**Comment by jalvesaq** [comment]
I finally got a Mac (for this weekend only). I didn't have any problem to build the current development version of vimcom, and I fixed all bugs that I have found when running MacVim. I couldn't build vim with X clientserver support, so this wasn't tested, but now the communication from vimcom to MacVim is working. I tested MacVim in three ways:

   - With `vimrplugin_applescript = 1` it runs R.app. The plugin is fully functional.
   - With `vimrplugin_applescript = 0` it opens the default terminal and runs R. The plugin is fully functional.
   - With the Vim application that comes with MacVim running inside Tmux, the Tmux window is split in two panes and R runs in the lower pane. The communication from vimcom to Vim doesn't work because MacVim developers chose not to include the server side of the clientserver feature in the Vim application and the plugin runs with the limitations described at [r-plugin-nox].

The only advantage of running R instead of R.app that I see is the possibility of getting R output colorized by the *colorout* package.

If you prefer to run Vim and R inside Tmux, you will need either to build Vim and vimcom with the X clientserver feature (as you have done before) or ask MacVim developers to include the server side of the Cocoa clientserver feature in their Vim application.

[r-plugin-nox]: http://www.lepem.ufc.br/jaa/r-plugin.html#r-plugin-nox
